source/automated-deployment.rst

```diff
@@ -0,0 +1,84 @@
+Automating Deployment with Puppet
+=================================
+Definitions
+-----------
+* BEAMLINE_WORKSTATION
+    Any computer that beamline staff designate as a computer they want the
+    collection stack installed on
+* BEAMINE_SERVER
+    The main "compute" computer where jupyterhub will point to
+* IPYTHON_CONFIGURATION:
```

Comment:
I think this is something that we can expect the BL staff to learn how to keep track of.